3rdparty/python/user_reqs.lock

```diff
@@ -2408,8 +2408,8 @@
   "only_wheels": [],
   "overridden": [],
   "path_mappings": {},
-  "pex_version": "2.36.0",
-  "pip_version": "25.0",
+  "pex_version": "2.37.0",
```

Comment:
Shouldn't `requirements.txt` also be updated in this PR?

Reply:
whoops; fixed.

Would love to not have that source dep on pex, but I know that's a totally different can of worms.